reference/checkouts.v3.yml

```
@@ -7933,6 +7934,60 @@ paths:
                 meta: {}
       security:
         - X-Auth-Token: []
+  '/stores/{store_hash}/v3/checkouts/{checkout-id}/token':
+    parameters:
+      - schema:
+          type: string
+        name: store_hash
+        in: path
+        required: true
+      - schema:
+          type: string
+        name: checkout-id
+        in: path
+        required: true
+    post:
+      tags:
+        - Checkout Token
+      summary: Create Checkout Token
+      operationId: checkout-token
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  checkoutToken:
+                    type: string
+                    example: beb3590088be88f59ba980d54a68df655cd8a1052a3e9caf535f3f820146c226
+                  meta:
+                    type: object
+              examples:
+                example-1:
+                  value:
+                    data:
+                      checkoutToken: beb3590088be88f59ba980d54a68df655cd8a1052a3e9caf535f3f820146c226
+                      meta: {}
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                maxUses:
+                  type: number
+                  example: 1
+                ttl:
+                  type: number
+                  example: 9999999999999
```

```suggestion
                  example: 86400
```

Could we add a description for the field `ttl`? The number means how many seconds for the token alive and maximum is 2,592,000 (30 days).